packages/embark-coverage/src/@types/mock-fs/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'mock-fs' {
+  var x: any;
```

Comment:
This is annoying but I don't know how to get around it.

Reply:
Have you tried just declaring the module as 

```
declare module 'mock-fs' {}
```

If that doesn't do it, maybe we can at least do

```
declare module 'mock-fs' {
  export const mock: any;
}
```

Reply:
That worked, thank you! 🎉

Reply:
I went and refactored it out. Sorry :-(